axonserver/src/main/java/io/axoniq/axonserver/message/command/hashing/ConsistentHashRoutingSelector.java

```diff
@@ -0,0 +1,39 @@
+package io.axoniq.axonserver.message.command.hashing;
+
+import io.axoniq.axonserver.message.command.RoutingSelector;
+import io.axoniq.axonserver.message.command.hashing.ConsistentHash.ConsistentHashMember;
+
+import java.util.Optional;
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.function.Function;
+
+/**
```

Comment:
Nit: Class level and constructor javadoc is missing.

Reply:
Done